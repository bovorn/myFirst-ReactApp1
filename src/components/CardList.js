import React from 'react';
import Todo from './Todo';

export default function cardList({ todos }) {
    return (
        <ul>
            {
                todos.map( (todo, i) => { 
                   return <Todo key={todos[i].id} title={todos[i].title} completed={todos[i].completed}/>;
                })
            }
        </ul>
    )
}
