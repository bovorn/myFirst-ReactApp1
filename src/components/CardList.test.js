import renderer from 'react-test-renderer';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John wick',
            email: 'j.wick@example.com',
            username: 'johnwick'
        },
    ];
    const wrapper = renderer.create(<CardList robots={mockRobots} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
})
