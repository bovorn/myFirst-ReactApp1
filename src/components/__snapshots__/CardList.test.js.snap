// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <div
    className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        John wick
      </h2>
      <p>
        j.wick@example.com
      </p>
    </div>
  </div>
</div>
`;
